R/tidyverse.R

```diff
@@ -201,10 +201,13 @@ use_tidy_coc <- function() {
     ".github/CODE_OF_CONDUCT.md"
   )
 
-  todo("Don't forget to describe the code of conduct in your README.md:")
+  todo("Don't forget to describe the code of conduct in your README:")
   code_block(
-    "Please note that this project is released with a [Contributor Code of Conduct](.github/CODE_OF_CONDUCT.md).",
-    "By participating in this project you agree to abide by its terms."
+    paste0("Please note that the ", value(project_name()),
```

Comment:
Should we make this just "tidyverse packages"?

Reply:
In that case, would r-lib packages use the traditional `use_code_of_conduct()`?

Reply:
Oh good point - in that case maybe we should pull out the repeated code into a function?

(Another random thought: should `code_block()` use `glue()`?)

Reply:
> (Another random thought: should code_block() use glue()?)

YES. Will do.